projects/dotcms-field-elements/src/components/dot-date-time/dot-date-time.tsx

```diff
@@ -0,0 +1,209 @@
+import {
+    Component,
+    Element,
+    Event,
+    EventEmitter,
+    Listen,
+    Method,
+    Prop,
+    State
+} from '@stencil/core';
+import Fragment from 'stencil-fragment';
+import {
+    DotFieldStatus,
+    DotFieldStatusClasses,
+    DotFieldStatusEvent,
+    DotFieldValueEvent,
+    DotLabel
+} from '../../models';
+import { Components } from '../../components';
+import DotInputCalendar = Components.DotInputCalendar;
+import { getClassNames, getTagError, getTagHint, getTagLabel } from '../../utils';
+
+const DATE_REGEX = new RegExp('(19|20)\\d\\d-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])');
+const TIME_REGEX = new RegExp('^(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])$');
+
+interface FormattedDate {
+    date: string;
+    time: string;
+}
+
+@Component({
+    tag: 'dot-date-time',
+    styleUrl: 'dot-date-time.scss'
+})
+export class DotDateTimeComponent {
+    @Element() el: HTMLElement;
+    /** format should be year-month-day hour:minute:second e.g., 2005-12-01 15:22:00 */
+    @Prop({ mutable: true })
+    value: string;
+    @Prop() name: string;
+    @Prop() label: string;
+    @Prop() hint: string;
+    @Prop() required: boolean;
+    @Prop() requiredMessage: string;
+    @Prop() validationMessage: string;
+    @Prop() disabled = false;
+    /** format should be year-month-day hour:minute:second | year-month-day | hour:minute:second */
+    @Prop() min: string;
+    /** format should be year-month-day hour:minute:second | year-month-day | hour:minute:second */
+    @Prop() max: string;
+    /** two values separates by a space, first goes to date, second to time */
+    @Prop() step: string;
```

Comment:
why separated by `space`, I think, were splitting things with comma

Reply:
![give-that-man-2-beer-com-14299358](https://user-images.githubusercontent.com/751424/57461228-dc834600-7233-11e9-87bd-bde50a44dc50.png)